src/java/org/apache/cassandra/metrics/ChunkCacheMetrics.java

```diff
@@ -50,12 +50,14 @@ public ChunkCacheMetrics(ChunkCache cache)
     public void recordHits(int count)
     {
         hits.mark(count);
+        requests.mark(-requests.getCount());
```

Comment:
Should be `requests.mark(count)`. I would also update the `requests` before the `hits` and the `misses` to be more consistent with the way we do the things in `InstrumentingCache`